deploy/run_with_summon.sh

```diff
@@ -59,7 +59,7 @@ ssl_cert=$($cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESP
 
 export CONJUR_SSL_CERTIFICATE=$ssl_cert
 
-if [ "${DEV}" = "false"  ]; then
+if [[ "${DEV}" = "false" || "${HELM_TESTS}" = "true"  ]]; then
```

Comment:
this looks as something we won't be able to understand six months from now. why not being on DEV env is the same as running HELM tests?

Reply:
Fixed this one